guide/english/algorithms/algorithm-performance/index.md

```diff
@@ -3,20 +3,20 @@ title: Algorithm Performance
 ---
 
 ## Algorithm Performance
-To understand the performance of algorithms, first we need to know the big-O notation.
+To understand the performance of algorithms, first, we need to know the big-O notation.
```

Comment:
Articles are really supposed to be in the 2nd person, plus the following gets rid of the need for the extra comma.
```suggestion
To understand the performance of algorithms, you first need to know the big-O notation.
```